src/monodroid/jni/shared-constants.hh

```diff
@@ -39,6 +43,18 @@ namespace xamarin::android::internal
 		static constexpr char MONO_SGEN_SO[]      = "monosgen-2.0";
 		static constexpr char MONO_SGEN_ARCH_SO[] = "monosgen-" __BITNESS__ "-2.0";
 #endif
+
+#if __arm__
+		static constexpr char android_abi[] = "armeabi_v7a";
```

Comment:
Why can't this use `-`, so that `armeabi-v7a` is valid?

Reply:
Because `_` is what Android uses when generating split APKs: `split_config.armeabi_v7a.apk` is what they generate and we check that name against the list of APKs passed to runtime init